image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -150,34 +150,54 @@ object FileMetadataReader {
           Dimensions(width, height)
         }
 
+        case Some("image/tiff") => for {
+          exifDir <- Option(metadata.getFirstDirectoryOfType(classOf[ExifIFD0Directory]))
```

Comment:
Blech, this whole metadata lib needs a Scala wrapper.

Consider starting to push out interactions with that lib _elsewhere_, or even give it the typeclass treatment and abstract implementation away entirely.